docs/getting-started.md

```diff
@@ -167,15 +168,16 @@ are using, if you are using something else:
 make publish-component-containers CONTAINER_REPO=localhost:5000/ocurity/dracon
 ```
 
-\* Notice that the repo we are using is slightly different than the
+*Notice that the repo we are using is slightly different from the
```

Comment:
I think this was an attempt to use italic multiline but it was not rendering correctly. Now it does.